data/windows.yaml

```diff
@@ -2103,3 +2103,23 @@ urls:
 - 'https://www.symantec.com/security_response/writeup.jsp?docid=2012-020609-4221-99&tabid=2'
 - 'http://www.nirsoft.net/utils/winsock_service_providers.html'
 - 'https://msdn.microsoft.com/en-us/library/windows/desktop/ms739923(v=vs.85).aspx'
+---
+name: WindowsDNSNameServer
```

Comment:
The NS in DNS already refers to Name Sever, maybe rename this to: WindowsDNSSettings